language_detector/main.py

```diff
@@ -2,10 +2,24 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
+
+def detectInOneLanguage(text,lang):
```

Comment:
I agree with Adonias. This code as it is would return incorrect language in some cases. A better logic would be to count the number of common words from each language that appear in the given text and pick the language that has the highest count.